task/sast-unicode-check-oci-ta/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# sast-unicode-check-oci-ta task
+
+Scans source code for non-printable unicode characters in all text files.
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|CACHI2_ARTIFACT|The Trusted Artifact URI pointing to the artifact with the prefetched dependencies.|""|false|
+|FIND_UNICODE_CONTROL_ARGS|arguments for find-unicode-control command.|-p bidi -v -d -t|false|
+|FIND_UNICODE_CONTROL_GIT_URL|URL from repository to find unicode control.|https://github.com/siddhesh/find-unicode-control.git#c2accbfbba7553a8bc1ebd97089ae08ad8347e58|false|
+|KFP_GIT_URL|URL from repository to download known false positives files.|""|false|
+|PROJECT_NVR|Name-Version-Release (NVR) of the scanned project. It is used to find path exclusions (it is optional). |""|false|
```

Comment:
In discussions about other sast checks we agreed to use `PROJECT_NAME` and have it default to the component name.  At build-time, components won't have an NVR.

Ref: https://github.com/konflux-ci/build-definitions/pull/1437/files#diff-4b781f85768af90bc67dfb600a14fb9e1819077a3c0acca7ef553aa9b6107478R12